### ISSUE TITLE
Contrast ratio error, text element in modal window

### ISSUE THREAD

**Issue by Grandmagic13** [dequelabs/axe-core-gems#236]
```
Using:
- `axe-core-rspec` and `axe-core-capybara` version: 4.4.0
- Testing with chromedriver, Chrome Version 102.0.5005.115 (Official Build) (arm64)
```
I have a modal header with the color `#2d3b45` in front of a `#ffffff` background but for some reason the foreground color picked up by axe is something different (`#f1f2f3`). There is no complicated code logic involved, the test code opens the modal successfully after which the `be_axe_clean.according_to :wcag2aa` matcher is called to validate the page object. This test also worked in the past and the page design hasn't changed, meaning the page itself shouldn't produce failures, but after a ruby update we had to update the axe libraries as well, and we've been getting this error ever since. I've seen color contrast issues before related to axe-core, the latest that has been fixed on May 12. Version 4.4.0 was available long before that, so maybe a follow-up is in order on the axe-core-gems side to use the latest version of axe-core (4.4.2) ?
I get the error:
```
Selector: .css-vgidk1-modalHeader > h2
HTML: <h2 dir="ltr" class="css-5vody5-view-heading">Delete Media</h2>
Fix any of the following:
- Element has insufficient color contrast of 1.11 (foreground color: #f1f2f3, background color: #ffffff, font size: 21.0pt (28px), font weight: normal). Expected contrast ratio of 3:1
```

**Comment by Grandmagic13** [comment]
I was hoping that updating the axe-core dependency would solve the issue, but after it happened (https://github.com/dequelabs/axe-core-gems/pull/239) I tried out the new release (4.4.1) and I tried running the failing test only to receive the same kind of errors. I double-checked, gems were up to date in my environment. It's kind of frustrating.

**Comment by michael-siek** [comment]
Hi @Grandmagic13 it looks like when updating to axe-core-rspec 4.4.0 it included changes to update the version of axe-core to 4.4.0 which had significant color contrast fixes as well as feature that may have made changes to the results provided. If this is still an issue please feel free to reopen this ticket. 

Ref: https://github.com/dequelabs/axe-core/releases/tag/v4.4.0

**Comment by padmavemulapati** [comment]
Now  issues unable to reproduce. As per @michael-siek comment this issue will not be coming if axe-core-gems-api version updated to >4.4.0 version.
